pandas/core/series.py

```diff
@@ -3617,7 +3618,7 @@ def map(self, arg, na_action=None):
 
         Parameters
         ----------
-        arg : function, dict, or Series
+        arg : function, colletions.abc.Mapping subclass or Series
```

Comment:
I think could simply just use "mapping" in docstrings

Reply:
I am not sure about that, the other docstrings I've seen in pandas use the whole `collections.abc.Mapping` and I believe it's better for the users who aren't familiar with `collections.abc`. Having the whole path can help them search it, while just `Mapping` won't be as helpful.
You did make me notice I had a typo though 😅